core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -613,8 +611,8 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
     val consumer = TestUtils.createNewConsumer(TestUtils.getBrokerListStrFromServers(servers), groupId = group,
       securityProtocol = SecurityProtocol.PLAINTEXT, props = Some(consumerConfig))
     try {
-      consumer.subscribe(Pattern.compile(".*"), new NoOpConsumerRebalanceListener)
-      consumeRecords(consumer)
+      consumer.subscribe(singletonList(kafka.common.Topic.GroupMetadataTopicName), new NoOpConsumerRebalanceListener)
```

Comment:
It seems the test was trying to test the pattern subscription, so we probably want to keep that unchanged.